doc/user/bgp.rst

```diff
@@ -3887,6 +3887,10 @@ structure is extended with :clicmd:`show bgp [afi] [safi]`.
 
    EVPN prefixes can also be filtered by EVPN route type.
 
+.. clicmd:: show bgp l2vpn evpn route [detail] [type <macip|2|multicast|3|es|4|prefix|5>] self-originate [json]"
```

Comment:
Double quotes at the end not needed.

Reply:
done. Thank you!

Reply:
-.. clicmd:: show bgp l2vpn evpn route [detail] [type <macip|2|multicast|3|es|4|prefix|5>] self-originate [json]"
+.. clicmd:: show bgp l2vpn evpn route [detail] [type <ead|1|macip|2|multicast|3|es|4|prefix|5>] self-originate [json]